lib/editor/util/validation.js

```diff
@@ -100,6 +101,12 @@ export function validate (
         (indices.length > 1 ||
           (indices.length > 0 && entities[indices[0]].id !== entity.id))
       )
+      console.log(idList.length)
+      console.log({entity})
+      if (agencies.length > 1 && entity.agency_id === null) {
+        reason = 'Identifier is required if more than one agency exists'
+        return {field: name, invalid: isRequiredButEmpty, reason}
```

Comment:
TBH I'm not quite sure. What is the purpose of the invalid property?  I only see it being used in these return statements. Is it for error/debugging?  If I remove it I get the same result, and if I put false or true I get the same result. 

In this instance we are saying that if there are more than one agency and the field value is empty then return that field name, the invalid status which in this instance would be true that it is currently invalid?  Or is it false because it failed validation?